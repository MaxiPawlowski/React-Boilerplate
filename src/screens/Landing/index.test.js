import React from 'react';
import { shallow } from 'enzyme';
import { StyledApp as Landing } from "./index";

describe('App', () => {
  let wrapper;

  describe('when rendered', () => {
    beforeAll(() => {
      wrapper = shallow(<Landing />).dive();
    });

    it('should render correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
