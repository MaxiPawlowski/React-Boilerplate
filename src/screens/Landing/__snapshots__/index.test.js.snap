// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App when rendered should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
